src/pages/projectSettings/HeaderButtons.tsx

```diff
@@ -26,6 +26,11 @@ import { formToGqlMap } from "./tabs/transformers";
 import { FormToGqlFunction, WritableTabRoutes } from "./tabs/types";
 import { ProjectType } from "./tabs/utils";
 
+const defaultToRepoDisabled: Set<WritableTabRoutes> = new Set([
+  ProjectSettingsTabRoutes.Notifications,
+  ProjectSettingsTabRoutes.Plugins,
+]);
```

Comment:
possibly consider moving to `tabs/types`? That's where `readOnlyTabs` exist, which seems kind of similar

Reply:
I considered that, but I felt like this property is very specific to the component and won't be reused, so decided to include it in the component file itself. 😺